.travis.yml

```diff
@@ -1,18 +1,21 @@
 language: python
 sudo: false
 python:
-  - 2.6
   - 2.7_with_system_site_packages
+  - 3.4
```

Comment:
Travis infrastructure.  There's nothing that should stop it working on Python 3.3 I think, although if it did pose any difficulty then I'd be happy to drop support for 3.3 at this point.